### DISCUSSION TITLE
https://blog.tidy-intelligence.com/posts/dplyr-vs-pandas/

### DISCUSSION THREAD

**Discussion by giscus** [tidy-intelligence/blog#1] [Announcements]
# https://blog.tidy-intelligence.com/posts/dplyr-vs-pandas/

A comparison of R’s dplyr and Python’s pandas data manipulation packages

https://blog.tidy-intelligence.com/posts/dplyr-vs-pandas/

**Comment by coforfe** [comment]
Thanks for this review!

In Python, you have a very similar "dplyr" approach with "datar" library.  With that, moving from R to Python, with respect to data cleansing/processing is  inmediate.

Reference: https://github.com/pwwang/datar

Thanks again!
Carlos.

**Reply by christophscheuch** [reply]
Thank you for the suggestion, I did not know this library. I'll give it a shot soon 🙏
